Formula/cargo.rb

```diff
@@ -0,0 +1,27 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+class Cargo < EmacsFormula
+  desc "Emacs Minor Mode for Cargo, Rust's Package Manager."
+  homepage "https://github.com/kwrooijen/cargo.el"
+  url "https://melpa.org/packages/cargo-20160525.519.tar"
```

Comment:
Same as the comment I left on toml-mode; this will disappear when the next version appears on MELPA.  We need a tagged release from GitHub or ELPA.